src/main/java/uk/gov/pay/directdebit/notifications/services/UserNotificationService.java

```diff
@@ -14,7 +14,8 @@
 
 public class UserNotificationService {
 
-    private static final String PLACEHOLDER_STATEMENT_NAME = "THE-CAKE-IS-A-LIE";
+    public static final String PLACEHOLDER_STATEMENT_NAME = "THE-CAKE-IS-A-LIE";
```

Comment:
is this a good idea?

Reply:
@mrlumbu we didn't want to copy the value, that is why we made it `public`

Reply:
OK clarified where`THE-CAKE-IS-A-LIE` fits into the whole picture. Ultimately this will be supplied by adminusers. This will be cleaned up as part of service onboarding stories. Hopefully 😬